src/client/index.ts

```diff
@@ -0,0 +1,48 @@
+export interface Genre {
+  id: string;
+  name: string;
+  parentId: string;
+}
+
+export interface Podcast {
+  id: string;
+  rss?: string;
+  email?: string;
+  image: string;
+  title: string;
+  country?: string;
+  website?: string;
+  language?: string;
+  genreIds?: string[];
+  publisher: string;
+  url?: string;
+  description: string;
+  totalEpisodes?: number;
+  earliestPublishedAt?: Date;
+  latestPublishedAt: Date;
+}
+
+export interface Episode {
+  id: string;
+  audio: string;
+  image: string;
+  title: string;
+  publisher: string;
+  podcastId: string;
+  description: string;
+  publishedAt: Date;
+  url: string;
+  lengthInSeconds?: number
+  explicit?: boolean;
+}
+
+export interface PageResult<T> {
+  results: T[];
+  page: number;
+  perPage: number;
+}
+
+export interface PodcastClient {
+  searchPodcasts(term: string, page: number, perPage: number): PageResult<Podcast>;
+  getPodcastEpisodes(podcast: string, page: number, perPage: number): PageResult<Episode>;
```

Comment:
@guieiras 
Olhando as issues (#1, #5, #19) dessa versão MVP, acho que só iremos (além do `/search`) consumir o endpoint
 [`/podcasts`](https://www.listennotes.com/api/docs/#post-api-v2-podcasts), então poderiamos explodir e adaptar essa função para que ela fique mais semelhante à maneira como consumiremos o endpoint supracitado.

Estava pensando, por exemplo, em algo do tipo:

```suggestion
byIds(ids: string[]): Promise<PageResult<Podcast>>;
byItuneIds(ids: string[]): Promise<PageResult<Podcast>>;
byRssUrls(rssUrls: string[]): Promise<PageResult<Podcast>>;
```